src/libasr/codegen/wasm_to_x64.cpp

```diff
@@ -582,6 +582,8 @@ class X64Visitor : public WASMDecoder<X64Visitor>,
     void visit_F32Sqrt() { visit_F64Sqrt(); }
 
     void gen_x64_bytes() {
+        emit_elf64_header(m_a);
+
```

Comment:
All the changes in the files `wasm_to_x64.cpp`, `x86_assembler.cpp` and `x86_assembler.h` needs to be reverted.